pext/qml/main.qml

```diff
@@ -675,18 +675,18 @@ ApplicationWindow {
             }
 
             Menu {
-                title: qsTr("Automatic updates")
+                title: qsTr("New versions")
 
                 MenuItem {
                     objectName: "menuEnableUpdateCheck"
-                    text: qsTr("Automatically check for Pext updates")
+                    text: qsTr("Check for new Pext versions automatically")
                     checkable: true
                     visible: internalUpdaterEnabled
                 }
 
                 MenuItem {
                     objectName: "menuEnableObjectUpdateCheck"
-                    text: qsTr("Automatically update modules and themes")
+                    text: qsTr("Install new modules and themes automatically")
```

Comment:
To me, this implies other modules and themes (those that "just became available") instead of updating the existing ones.

Reply:
"Keep all modules and themes up to date?"

Reply:
I'm still not completely convinced. Besides, merging it now will break all existing translations, and waiting for translations to be completed is already delaying every release so far. I'm willing to discuss further though, but 0.16 will ship with the strings as they are now.